crates/assists/src/handlers/extract_module_to_file.rs

```diff
@@ -0,0 +1,94 @@
+use ide_db::source_change::FileSystemEdit;
+use vfs::FileId;
```

Comment:
```suggestion
use base_db::FileId;
```

should work, and they you don't need a vfs dep